File: stats/updates/entry.cs
(doc,req) ->
  mustache = require("vendor/couchapp/lib/mustache")
  ddoc = this;
  	
  doc.moderator = req.form.moderator
  
  doc.teamAscore = (Number req.form.teamAscore)
  doc.teamBscore = (Number req.form.teamBscore)
  
  doc.teamAtuh = (Number req.form.teamAtuh)
  doc.teamBtuh = (Number req.form.teamBtuh)
  
  doc.teamAbhrd = (Number req.form.teamAbhrd)
  doc.teamBbhrd = (Number req.form.teamBbhrd)
  
  doc.teamAbpts = (Number req.form.teamAbpts)
  doc.teamBbpts = (Number req.form.teamBbpts)

  for player,i in doc.teamAstats
    player.tens = (Number req.form["teamA#{i}tens"])
    player.negs = (Number req.form["teamA#{i}negs"])
    player.tuh = (Number req.form["teamA#{i}tuh"])
  
  for player,i in doc.teamBstats
    player.tens = (Number req.form["teamA#{i}tens"])
    player.negs = (Number req.form["teamA#{i}negs"])
    player.tuh = (Number req.form["teamA#{i}tuh"])

  
  resp = "<html><title>redirect</title><body><a href='../../_show/game_entry/#{doc._id}'>Click Here</a><br>
  <a href='../../_list/games/games_by_bracket'>Game List</a></body></html>"
  
  [doc,resp]

File: stats/updates/entry.js
function(doc,req) {
	var mustache = require("vendor/couchapp/lib/mustache")
  var ddoc = this;
	var resp
		
	doc.moderator = req.form.moderator

	doc.teamAscore = parseInt(req.form.teamAscore)
	doc.teamBscore = parseInt(req.form.teamBscore)
	
	doc.teamAtuh = parseInt(req.form.teamAtuh)
	doc.teamBtuh = parseInt(req.form.teamBtuh)
	
	doc.teamAbhrd = parseInt(req.form.teamAbhrd)
	doc.teamBbhrd = parseInt(req.form.teamBbhrd)

	doc.teamAbpts = parseInt(req.form.teamAbpts)
	doc.teamBbpts = parseInt(req.form.teamBbpts)
	
	for (var i in doc.teamAstats) {
		doc.teamAstats[i].tens = parseInt(req.form["teamA"+i+"tens"])
		doc.teamAstats[i].negs = parseInt(req.form["teamA"+i+"negs"])
		doc.teamAstats[i].tuh = parseInt(req.form["teamA"+i+"tuh"])
	}
	
	for (var i in doc.teamBstats) {
		doc.teamBstats[i].tens = parseInt(req.form["teamB"+i+"tens"])
		doc.teamBstats[i].negs = parseInt(req.form["teamB"+i+"negs"])
		doc.teamBstats[i].tuh = parseInt(req.form["teamB"+i+"tuh"])
	}
	
	resp = '<html><title>redirect</title><body><a href="../../_show/game_entry/'+doc._id+'">Click Here</a></body></html>'
	
	return [doc,resp]
}